coordinator/internal/logic/libzkp/Makefile

```
@@ -1,16 +1,18 @@
+include ../../../../build/common.mk
+
 .PHONY: help fmt clippy test test-ci test-all
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)